pages/api/availability/week.ts

```diff
@@ -0,0 +1,30 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getSession } from 'next-auth/client';
+import prisma from '../../../lib/prisma';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const session = await getSession({req: req});
+
+  if (!session) {
+    res.status(401).json({message: "Not authenticated"});
+    return;
+  }
+
+  if (req.method == "PATCH") {
+
+    const startMins = req.body.start;
+    const endMins = req.body.end;
+
+    const updateWeek = await prisma.schedule.update({
+      where: {
+        id: session.user.id,
+      },
+      data: {
+        startTime: startMins,
+        endTime: endMins
+      },
+    });
+
+    res.status(200).json({message: 'Start and end times updated successfully'});
```

Comment:
Codacy has a fix for the issue: [Replace `message:·'Start·and·end·times·updated·successfully'` with `·message:·"Start·and·end·times·updated·successfully"·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
    res.status(200).json({ message: "Start and end times updated successfully" });
```